biome.json

```diff
@@ -0,0 +1,34 @@
+{
+	"$schema": "https://biomejs.dev/schemas/2.0.6/schema.json",
+	"vcs": {
+		"enabled": false,
+		"clientKind": "git",
+		"useIgnoreFile": false
+	},
+	"files": {
```

Comment:
I guess we've to include folders we wish to apply formatting on here
Cos if the files were included then, they would've been formatted from your commit!!
I think the files are not parsed for formatting.